R/correctionsataglance-data.R

```diff
@@ -203,7 +203,7 @@ parseCorrThresholds <- function(reportObject, timezone){
       type <- rep(threshold[['type']], times=nrow(periods))
       startDates <- periods[['startTime']]
       endDates <- periods[['endTime']]
-      value <- periods[['referenceValue']]
+      value <- if(isEmptyOrBlank(periods[['referenceValue']])){NA} else{periods[['referenceValue']]}
```

Comment:
generally when you use the `if() {} else {}` syntax, you don't want it in a single line. So you would need:

```r
if(isEmptyOrBlank(periods[['referenceValue']]) {
    value <- NA
} else {
    value <- periods[['referenceValue']]
}
```

If you know that `periods[['referenceValue']]` is only ever going to have one value in it, you can do 
```r
value <- ifelse(isEmptyOrBlank(periods[['referenceValue']]), NA, periods[['referenceValue']])
```